running/README.md

```diff
@@ -590,6 +590,49 @@ This can be achieved by setting the `max_jobs` property of a partition in the `s
 
 ```
 
+## Execution modes
+
+Reframe 2.5 permits you to define different *execution modes* of the framework.
+An execution mode is merely a set of predefined command-line options that will be passed to Reframe when this mode is invoked.
+You can define execution modes per system in the Reframe's configuration file.
+For example, one could define a global `maintenance` execution mode as follows in the `settings.py` file:
+
+```python
+site_configuration = ReadOnlyField({
+    'systems' : {
+        ...
+    },
+    'environments' : {
+        ...
+    },
+    'modes' : {
+        *' : {
+            'maintenance' : [
+                '--exec-policy=async',
+                '--output=$APPS/UES/$USER/regression/maintenance',
```

Comment:
For the first part, yes. For the second part I think it's good to have an environment variable somehow, so that we show that they are treated correctly. We don't say it in the text, but if they are there they, it is at least implied. I could though add it explicitly to the text.